pkgs/by-name/_6/_64gram/package.nix

```diff
@@ -0,0 +1,28 @@
+{ lib
+, fetchFromGitHub
+, telegram-desktop
+}:
+
+telegram-desktop.overrideAttrs (old: rec {
+
+  pname = "64Gram";
+  version = "1.1.15";
+
+  src = fetchFromGitHub {
+    owner = "TDesktop-x64";
+    repo = "tdesktop";
+    rev = "v${version}";
+
+    fetchSubmodules = true;
+    hash = "sha256-3HLRv8RTyyfnjMF7w+euSOj6SbxlxOuczap5Nlizsvg=";
+  };
+
+  meta = with lib; {
+    description = "An unofficial Telegram Desktop providing Windows 64bit build and extra features";
+    license = licenses.gpl3Only;
+    platforms = platforms.all;
+    homepage = "https://github.com/TDesktop-x64/tdesktop";
+    changelog = "https://github.com/TDesktop-x64/tdesktop/releases/tag/v${version}";
+    maintainers = with maintainers; [ clot27 ];
```

Comment:
```suggestion
    maintainers = with maintainers; [ clot27 ];
    mainProgram = "telegram-desktop";
```

```
=> nix run github:clot27/nixpkgs/init-64gram#_64gram
error: unable to execute '/nix/store/6b464aqwsb2aji9jl7kxq3mdbd8rll50-64Gram-1.1.15/bin/64Gram': No such file or directory
```

Reply:
And mark as broken on darwin since it doesn't build there.

Reply:
@wegank How to do that?

Reply:
Done!